src/components/internals/calendar-menu/calendar-menu.mod.css

```diff
@@ -0,0 +1,27 @@
+.menuBase {
+  overflow-y: scroll;
+  border: 1px solid var(--token-border-color-input-focus);
+  border-radius: var(--token-border-radius-input);
+  margin-top: 4px;
+  position: absolute;
+  width: 100%;
+  background-color: white;
+}
+
+.menuBase.warning {
+  border-color: var(--token-border-color-input-warning);
+}
+
+.menuBase.error {
+  border-color: var(--token-border-color-input-error);
+}
+
+.menu {
+  composes: menuBase;
+  padding-bottom: 10px;
```

Comment:
Is this a sum of 4 and 6 on the `.container`? Should we have variables and a `calc` here?

Reply:
It's not the sum of anything to my knowledge. We don't need to have a design token for everything, it's okay if edge-cases use their own values. A date-calendar is a one-off component imho. There is no token for a spacing of 10 either, so I hardcoded it.